modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2779,13 +2779,15 @@ void ONNXImporter::parseResize(LayerParams& layerParams, const opencv_onnx::Node
     if (layerParams.has("coordinate_transformation_mode"))
     {
         String interp_mode = layerParams.get<String>("coordinate_transformation_mode");
-        CV_Assert_N(interp_mode != "tf_crop_and_resize", interp_mode != "tf_half_pixel_for_nn");
+        CV_Assert_N(interp_mode != "tf_crop_and_resize");
```

Comment:
`CV_Assert_N` is deprecated, see this comment https://github.com/opencv/opencv/pull/22898#discussion_r1045792566